mne/io/pick.py

```diff
@@ -116,9 +123,9 @@ def channel_type(info, idx):
     type : str
         Type of channel. Will be one of::
 
-            {'grad', 'mag', 'eeg', 'stim', 'eog', 'emg', 'ecg', 'ref_meg',
-             'resp', 'exci', 'ias', 'syst', 'misc', 'seeg', 'bio', 'chpi',
-             'dipole', 'gof', 'ecog', 'hbo', 'hbr'}
+            {'grad', 'mag', 'eeg', 'csd', 'stim', 'eog', 'emg', 'ecg',
```

Comment:
@larsoner csd is a coil type and not a channel type so I am wondering if we should not have eeg accept a string 'csd' to match MEG behavior. I am thinking out loud. I am sure you have thought about this more than me.

Reply:
We could probably do this instead. It just makes for weird scenarios where you would do `eeg='eeg'` to ensure you have actual EEG channels and not EEG-CSD channels. But it is clearer in terms of how we treat MEG and fNIRS channels (at least aside from hbo/hbr channels).

Reply:
forget it. It's fine as it is. Doing eeg='eeg' does not seem a good idea.